src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/appointments/AppointmentService.kt

```diff
@@ -172,6 +172,10 @@ private fun mapModel(entity: OffenderIndividualSchedule): AppointmentResponse =
     status = entity.eventStatus.code,
     subtype = entity.eventSubType.code,
     internalLocation = entity.internalLocation?.locationId,
-    prisonId = entity.prison?.id,
+    prisonId = entity.prison?.id ?: entity.toPrison?.id,
```

Comment:
In a few cases in the existing data the prison is null but the 'to_prison' is populated